share/spice/twitch_streams/twitch_streams.js

```diff
@@ -0,0 +1,69 @@
+(function (env) {
+    "use strict";
+    env.ddg_spice_twitch_streams = function(api_result){
+
+        if (api_result.error) {
+            return Spice.failed('twitch_streams');
+        }
+        
+        var streams = [];
+        if (!($.isArray(api_result.streams))) {
+            streams = [api_result.streams];
+        } else {
+            streams = api_result.streams;
+        }  
+        
+        //Get query, split into an array on space, append query to the end for moreat
+        var query = DDG.get_query().split(" ");
+        var moreAt = '';
+        for (var i = 2; i < query.length; i++) {
+            moreAt = moreAt + query[i];
+            if(i != query.length - 1){
+                moreAt += '+';
+            }
+        }
```

Comment:
You can get the query using `script = $('[src*="/js/spice/twitch_streams/"]')[0]` as oppose to `DDG.get_query()` See documentation here https://duck.co/duckduckhack/spice_advanced_frontend